src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -25,6 +25,11 @@ def test_get_property(device):
 
 def test_get_runtime_model(device):
     compiled_model = generate_relu_compiled_model(device)
+
+    core = Core()
+    user_stream = compiled_model.export_model()
+    core.import_model(user_stream, device)
```

Comment:
it's totally unclear why do we need to export / import model to resolve a segfault

Reply:
@siddhant-0707 mentioned that it is unclear why. However, my best bet would be creating actual `core = Core()` that is somehow affecting internals. Would you be able to test it out?

Reply:
`compiled_model` (returned by `generate_relu_compiled_model`) is expected to hold some internal to prolong its live time and ensure correct destruction. 
But if it's 100% reproducible, it can be debugged with dbg. @siddhant-0707 could you please try to collect debugger logs?